lib/domain/entity/component/request_component.dart

```diff
@@ -349,11 +351,13 @@ class RequestComponent with _$RequestComponent {
         codeLines.add(
           e.getParamBodyDeclaration(
             fileType,
-            false,
+            isRequiredRequestBody: false,
           ),
         );
       }
     }
+
+    codeLines.sort((a, b) => b.startsWith('required') ? 1 : -1);
```

Comment:
@cozvtieg9, as we discussed today, I've added logic to sort parameters in methods